localstack/services/lambda_/invocation/docker_runtime_executor.py

```diff
@@ -358,10 +358,11 @@ def start(self, env_vars: dict[str, str]) -> None:
                 self.container_name, f"{str(get_runtime_client_path())}/.", "/"
             )
             # tiny bit inefficient since we actually overwrite the init, but otherwise the path might not exist
-            if config.LAMBDA_INIT_DEBUG:
+            if config.LAMBDA_INIT_BIN_PATH:
```

Comment:
(already hit me 😅 when trying to test the changes while having `LAMBDA_INIT_BIN_PATH` set by default)

Reply:
The lambda init path was necessary anyway before. Now you can set only the init path to enable usage of a new lambda init binary without having to setup debugging itself.
You cannot toggle the init copying with the debug variable, but I would suggest using different profiles for this, instead of the toggle of init debug?

I can leave it out from this PR if you do not think it improves usability for developers, but for me it simplifies it, because I more often try out new inits without debugging than with.

Reply:
I agree that we should have a way to simplify init testing without debugging 👍 

Using profiles makes a lot of sense here 💯 . Then having two `entrypoint.sh` versions, one for testing (`exec /var/rapid/init`) and one for debugging makes switching easier. Thanks for sharing 👍